modules/cache/src/main/java/org/eclipse/xpanse/modules/cache/RegionAzCacheConfig.java

```diff
@@ -28,13 +33,16 @@ public class RegionAzCacheConfig {
      *
      * @return CaffeineCacheManager
      */
-    @Bean
-    public CaffeineCacheManager caffeineCacheManager() {
-        CaffeineCacheManager cacheManager = new CaffeineCacheManager(
-                CacheNames.REGION_AZ_CACHE_NAME);
+    @Primary
```

Comment:
can we remove the `@primary` annotation here? this can have some unexpected issues.

Reply:
Removing the `@primary`  will cause the following exception to be thrown when the application is starting:
![idea64_zgxiNz1lIh](https://github.com/eclipse-xpanse/xpanse/assets/121531362/2995130e-c2aa-45f7-8473-fb198120be64)

Reply:
No. Removing the @primary will cause exception as following:
![Uploading idea64_zgxiNz1lIh.png…]()

Reply:
No. Removing the @primary will cause the exception as below:
![image](https://github.com/eclipse-xpanse/xpanse/assets/121531362/62cc422b-03eb-4efc-b3a9-d6952f9f9db8)
I tried to add a special name to credential cache management, but it still failed until the @primary was used.